src/Server/Resource/DoctrineResource.php

```diff
@@ -348,6 +348,11 @@ public function create($data)
 
         $this->getObjectManager()->flush();
 
+        $results = $this->triggerDoctrineEvent(DoctrineResourceEvent::EVENT_CREATE_FLUSH, $entity, $data);
+        if ($results->last() instanceof ApiProblem) {
```

Comment:
It's true events were not handled as they should and that's why they were reworked.

Will you modify the README.md and explain _FLUSH events with emphasis on why an ApiProblem could be returned even after the entity in question has been persisted?

FYI although I haven't used it I understand there's a file upload already in place in Apigility.